service/src/main/resources/db/migration/V373__calendar_event_time_reservation.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE calendar_event_time
+(
+    id                uuid PRIMARY KEY                  DEFAULT ext.uuid_generate_v1mc(),
+    created           timestamp with time zone NOT NULL DEFAULT now(),
```

Comment:
I think this should be rename to `created_at`. AFAIK these new updated & created timestamp field names were agreed during some tech discussions:
<img width="607" alt="image" src="https://github.com/espoon-voltti/evaka/assets/886091/cf359aa6-2076-4882-aa74-0b4550c1095e">

Reply:
Fixed